performancecopilot/pcp
# pmlogger rc script, qa

Changes committed to git@github.com:kmcdonell/pcp.git 20200418

Ken McDonell (4):
      qa/common.check: fix diagnostic typo in _service()
      src/pmlogger/rc_pmlogger: important fix after SIGUSR2 for log rolling
      qa/348: important fixes
      qa/297.out: remade after pmGetContextHostName_r -> pmGetContextHostName_rr change

 qa/297.out               |    4 ++--
 qa/348                   |    7 +++++--
 qa/common.check          |    2 +-
 src/pmlogger/rc_pmlogger |    2 +-
 4 files changed, 9 insertions(+), 6 deletions(-)

Details ...

commit 8c780eecb567a986ad5e6e6bec43668a39c43dc1
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Sat Apr 18 19:10:16 2020 +1000

    qa/297.out: remade after pmGetContextHostName_r -> pmGetContextHostName_rr change

commit b9243478014e8ef9ec4bf1db7199bce7e5f2d994
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Sat Apr 18 18:14:47 2020 +1000

    qa/348: important fixes
    
    1. the Bourne shell is in /bin/sh, not /usr/bin/sh ... I'm getting really
       tired of this recurring problem ... please use qa/new to create new
       QA tests and avoid this (and a raft of other problems).
    
    2. /sys/kernel/security/lockdown does not always exist ... check before
       trying to feed this file to fgrep

commit e24e498803709339f945ac45f23ac012b13daee1
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Sat Apr 18 16:08:59 2020 +1000

    src/pmlogger/rc_pmlogger: important fix after SIGUSR2 for log rolling
    
    For systems without systemd, the /etc/init.d/pmlogger script has to
    be able to identify pmlogger instances started by pmlogger_check and
    pmlogger_daily ... this was not happening correctly after the nightly
    log rotation and re-exec of pmlogger.

commit 9b0d191cecda71b518981216970eb9ca17166518
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Sat Apr 18 16:07:21 2020 +1000

    qa/common.check: fix diagnostic typo in _service()
